03-Raster-Calculations-In-R.Rmd

```diff
@@ -96,24 +109,15 @@ buildings, etc with the influence of ground elevation removed.
 <figure>
     <a href="{{ site.baseurl }}/images/raster_timeseries/lidarTree-height.png">
     <img src="{{ site.baseurl }}/images/raster_timeseries/lidarTree-height.png"></a>
-    <figcaption>A canopy height model represents the difference between the digital 
-    surface model and a digital terrain model. It represents the actual height 
-    of the trees with the influence of elevation, removed.</figcaption>
+    <figcaption> A canopy height model represents the difference between the
+    digital surface model and a digital terrain model. It represents the actual
+    height of the trees with the influence of elevation removed. Image source:
+    NEON, Inc.</figcaption>
```

Comment:
Lucky for us, we have "find in all files" SO if we are consistent with our Source text, it will be fairly easy to rebrand.
Although maybe we just write NEON now? I'll run this by Wendy. We can brand it to the NEON project. That is better in my opinion anyway! :)